CHANGELOG/CHANGELOG-3.2.md

```diff
@@ -0,0 +1,77 @@
+# Release notes for v3.2.0
+
+[Documentation](https://docs.k8s.io/docs/home)
+# Changelog since v3.1.0
+
+## Changes by Kind
+
+### Feature
+
+- Add configurable throughput parameters for clients to API server ([#286](https://github.com/kubernetes-csi/external-attacher/pull/286), [@RaunakShah](https://github.com/RaunakShah))
+- Bump csi translation lib version ([#293](https://github.com/kubernetes-csi/external-attacher/pull/293), [@andyzhangx](https://github.com/andyzhangx))
+- Set the value of "migrated" field in the metrics to true or false to indicate if the call is a migration enabled feature or not ([#292](https://github.com/kubernetes-csi/external-attacher/pull/292), [@nearora-msft](https://github.com/nearora-msft))
+
+### Bug or Regression
```

Comment:
Is https://github.com/kubernetes-csi/external-attacher/pull/299 worth a release note?